### PR TITLE
Update Validatingphonenumbers.py

### PR DESCRIPTION
**Pull request by HighnessAtharva**
Updated Code to Python3 standards.

### CODE REVIEW

File: RegexandParsing/Validatingphonenumbers.py [hunk 1]
```diff
@@ -4,19 +4,20 @@
 Domain    : Python
 Author    : Ahmedur Rahman Shovon
 Created   : 15 July 2016
+Updated   : 3 April 2021
 Problem   : https://www.hackerrank.com/challenges/validating-the-phone-number/problem
 '''
 # Enter your code here. Read input from STDIN. Print output to STDOUT
-n=int(raw_input())
+n=int(input())
```
Comment (arsho) [inline RegexandParsing/Validatingphonenumbers.py:11]: @HighnessAtharva ,

The following code is using regex. Please update the PR with this:

```
from re import compile, match


n = int(input())
for _ in range(n):
    phone_number = input()
    condition = compile(r'^[7-9]\d{9}$')
    if bool(match(condition, phone_number)):
        print("YES")
    else:
        print("NO")
```

**Review by arsho — CHANGES_REQUESTED** [review]
Please accommodate the review comment.

File: RegexandParsing/Validatingphonenumbers.py (hunk 1 above)
Comment (HighnessAtharva) [reply RegexandParsing/Validatingphonenumbers.py]: Done.

**Review by arsho — APPROVED** [review]
Splendid!

**Pull request merged**